### PR TITLE
Fix javadoc errors

### PR DESCRIPTION
**Pull request by laeubi**
sadly javadoc is still not part of the verification steps :-\
And eclipse does not complain even I have enabled malformed javadoc->error

### CODE REVIEW

File: bundles/org.eclipse.core.resources/src/org/eclipse/core/resources/IWorkspace.java [hunk 1]
```diff
@@ -81,13 +81,12 @@
  * <li>Helper classes like ServiceCaller:
  *
  * <pre>
- * ServiceCaller.callOnce(getClass(), IWorkspace.class, workspace -> {
+ * ServiceCaller.callOnce(getClass(), IWorkspace.class, workspace -&gt; {
  * 	// do something with the workspace
  * });
  * </pre>
```
Comment (iloveeclipse) [inline bundles/org.eclipse.core.resources/src/org/eclipse/core/resources/IWorkspace.java:87]: Isn't </li> missing here?

**Review by iloveeclipse — COMMENTED** [review]
I haven't tried that in IDE, the changes seem to be correct, one tag is missing still

**Review by iloveeclipse — APPROVED** [review]

**Comment by laeubi** [comment]
Thanks for helping with this!

**Comment by mickaelistria** [comment]
By the way, I'd like to eventually have those errors automatically checked and reported on CI. Would you mind reporting a bug about it so the idea doesn't get lost?

**Comment by laeubi** [comment]
@mickaelistria see https://bugs.eclipse.org/bugs/show_bug.cgi?id=570430

**Pull request merged**